### ISSUE TITLE
Spike: Can we have both routing modes?

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#15]
### Expected result

Can we have both the [content tree-based](https://docs.kentico.com/13/developing-websites/routing/content-tree-based-routing) and the [traditional ASP.NET routing](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing) side by side in one project?

Can we model the result of tree-based routing with traditional routes without a huge amount of custom coding? (E.g. with lots of [route constraints](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-3.1#route-constraint-reference))

### Additional context

The ideal scene is to demonstrate both approaches as equivalents. The solution would be configured to primarily use the tree-based mechanism, with the traditional one as either a fallback (in the request pipeline) or as a commented-out alternative. The two approaches would be explained in two course pages. It would be up to the trainee to choose one over another.

### Resources

* URL

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
